### PR TITLE
Creating new hug and fight lines

### PR DESCRIPTION
**Pull request by blastoise186**
# Please check the following boxes

- [x] I've read and agree to the [Contribution Guidelines](https://github.com/Dougley/MBv2/blob/master/.github/CONTRIBUTING.md) and to the [Code of Conduct](https://github.com/Dougley/MBv2/blob/master/.github/CODE_OF_CONDUCT.md)
- [x] I've checked that I didn't commit sensitive information like tokens or other login information
- [x] I haven't hardcoded any IDs or anything else that shouldn't be (with the exception being permissions)
- [x] I tested my code and I verified it's working to the best of my ability
- [ ] I checked if my code doesn't violate the styleguide with `npm test` - N/A due to Travis CI and the fact I'm only adding new hug and fight lines

# Describe your pull request

With Dabbit and Danny switching places, I'd like to add new hug and fight lines to recognise them with.

### CODE REVIEW

**Comment by blastoise186** [comment]
Ok, that didn't quite work out. Lets try that again. Give me a moment

**Comment by zSnails** [comment]
alright

**Comment by blastoise186** [comment]
I'm a bit confused as to whether that worked out or not. Could someone check?

File: src/commands/fight.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const templates = [
   '{s} begged their pet to attack {r}, but the pet stared back with no indication of understanding.',
   '{s} fought like a dog, but {r} fought back like a bear, winning the fight!',
   'A wild {r} appears!\n{r} uses Bite! It\'s not very effective...\n{s} uses Mega Punch! It\'s very effective!\n{s} has won!',
-  'As {s} ran all sweaty and tired reaching out for a last punch, {r} dashed to the side, leaving {s} tumbling onto the ground.'
+  'As {s} ran all sweaty and tired reaching out for a last punch, {r} dashed to the side, leaving {s} tumbling onto the ground.',
+  '{s} tried to modify the Blastoise Dupe Bomber 3000 to take down {r} with tons of dupe reports, but Dannysaur got there first and denied them all... Which broke the machine.'
```
Comment (Dougley) [inline src/commands/fight.js]: This is a little self-indulgent, could we make this a bit more general?

Comment (blastoise186) [reply src/commands/fight.js]: done

**Comment by blastoise186** [comment]
Thanks 👍

**Pull request merged**